Documentation/ReleaseProcess.md

```diff
@@ -0,0 +1,83 @@
+# GitHub release configuration
```

Comment:
Under this can you include a new subtitle h2 "## Generate automatic release notes using Release Drafter" (and then also move the other subtitles one level down)?
This will allow us to update this document in the future with other details regarding our release process.

Reply:
That makes sense, I've updated to include the header you suggested and shifted the children headers accordingly